module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,27 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+#include <string>
+#include <vector>
+
+void removeVowels(std::vector<std::string>& words) {
+
+    for (auto &word : words) {
+        std::string temp;
+        for (auto it = begin(word); it != end(word); it++) {
+            if (!isVowel(*it)) {
+                temp.push_back(*it);
+            }
+        }
+        word.swap(temp);
+    }
+}
+
+bool isVowel(const char& c) {
+    std::array<char, 12> vowels {'a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U', 'y', 'Y'};
```

Comment:
You definitely should make it constexpr and I would consider moving it to the header.